tests/app/test_jinja_filters.py

```diff
@@ -762,3 +763,27 @@ def test_get_answer_label_with_no_answer_label_and_question_titles(self):
 
         # Then
         self.assertEqual(answer_label, 'default_question_title')
+
+
+    @patch('app.jinja_filters.format_unordered_list')
+    def test_format_repeated_summaries_unformatted(self, patched_format): # pylint: disable=no-self-use
```

Comment:
Can you create another test with a different delimiter eg a comma with an address.
Also test with some empty values to check we don't get `51 my road, , ,Bristol` etc

Reply:
Good point. Should be sorted.